.github/workflows/deploy_snapshot.yml

```diff
@@ -103,6 +103,9 @@ jobs:
       - name: Copy checksum files
         run: aws s3 cp image-digest.txt  s3://${{ secrets.AWS_BUCKET }}/${{ env.S3_FOLDER }}-${{ steps.slug.outputs.sha7 }}/image-digest.txt
 
+      - name: Copy checksum files (new)
```

Comment:
Should we re-instate https://github.com/dockstore/dockstore/commit/098dfa5149792d51d65323eee6162e8b5546ae54 and modify https://github.com/dockstore/workflow-actions/blob/124500e24cfae09a87b12f1409ea288235530563/.github/workflows/deploy_image.yaml#L100-L101 instead?

Reply:
Medium term, yes. 

This is more for super short term. Like if we need a 1.16.6 next week kinda thing.